### PR TITLE
wallet-rpc: added IPs to error logging in simple_http_connection_handler

### PR DESCRIPTION
**Pull request by jcktm**
This implements #3127

### CODE REVIEW

File: contrib/epee/include/net/http_protocol_handler.h [hunk 1]
```diff
@@ -196,7 +196,7 @@ namespace net_utils
 				response.m_response_comment = "OK";
 				response.m_body.clear();
 
-				return m_config.m_phandler->handle_http_request(query_info, response, m_conn_context);
+				return m_config.m_phandler->handle_http_request(query_info, response, this->m_conn_context);
```
Comment (moneromooo-monero) [inline contrib/epee/include/net/http_protocol_handler.h]: Is this needed ?

File: contrib/epee/include/net/http_protocol_handler.inl [hunk 2]
```diff
@@ -268,7 +269,7 @@ namespace net_utils
 					m_is_stop_handling = true;
 					if(m_cache.size() > HTTP_MAX_URI_LEN)
 					{
-						LOG_ERROR("simple_http_connection_handler::handle_buff_out: Too long URI line");
+						LOG_ERROR("simple_http_connection_handler::handle_buff_out: Too long URI line [" << this->m_conn_context << "]");
```
Comment (moneromooo-monero) [inline contrib/epee/include/net/http_protocol_handler.inl]: Please use the existing macros for this, such as LOG_ERROR_CC. That way the output's all consistent (and it's easier to write too).

Comment (jcktm) [reply contrib/epee/include/net/http_protocol_handler.inl]: Fixed

File: contrib/epee/include/net/http_protocol_handler.h (hunk 1 above)
Comment (moneromooo-monero) [reply contrib/epee/include/net/http_protocol_handler.h]: Thanks, I did not know about this.

**Comment by luigi1111** [comment]
@jcktm please rebase.

**Comment by jcktm** [comment]
Rebased

**Pull request merged**